src/lib/config.js

```diff
@@ -1,5 +1,12 @@
 const RC = require('parse-strings-in-object')(require('rc')('MLAPI', require('../../config/default.json')))
+const fs = require('fs')
 
+function getFileContent (path) {
+  if (!fs.existsSync(path)) {
+    throw new Error('File doesn\'t exist')
```

Comment:
@shashi165 There is no way of knowing how upstream consumers of this module will handle the throw...I know there is no logger in context at this point, but if I were you I would spit out a console.log with the expected file path/name and the fact that it is not found. In that case, even if the consumer barfs without mentioning the cause, a human will be able to see (unless in the unlikely event stdout is redirected) what potentially caused a problem.

Reply:
Is it better to make a major version bump than risk people wasting a lot of time checking logs?

Reply:
Besides, if they've requested JWS signing and misconfigured the service, and it just carries on running and prints some log message, that's potentially a _much_ bigger risk than crashing.

Reply:
@msk- you are missing the point I think. I am not suggesting to not throw here. I am suggesting to write some log output BEFORE throwing. This covers cases where, at some point in the future, some consumer of this module may swallow this exception.

Reply:
@bushjames Added a console.log before throw

Reply:
James, I was missing the point, you were correct.